### PR TITLE
Refactor cli/test_ping for pytest

### PR DESCRIPTION
**Pull request by mshriver**
```
setup@localhost:~/repos/robottelo (pytest-ping-13989$%<>) % pytest -vx tests/foreman/cli/test_ping.py
======================================================================================== test session starts =========================================================================================
platform linux -- Python 3.8.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /home/setup/repos/robottelo/.robottelo/bin/python3.8
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/setup/repos/robottelo, configfile: pyproject.toml
plugins: forked-1.3.0, ibutsu-1.14.1, xdist-2.2.1, mock-3.5.1, reportportal-5.0.8, cov-2.11.1, services-2.2.1
collected 1 item                                                                                                                                                                                     

tests/foreman/cli/test_ping.py::test_positive_ping PASSED                                                                                                                                      [100%]

========================================================================================= 1 passed in 17.40s =========================================================================================

```

### CODE REVIEW

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ping.py

File: tests/foreman/cli/test_ping.py [hunk 1]
```diff
@@ -12,57 +12,47 @@
...
+           != 0
 
-        :steps:
-            1. Execute hammer ping
-            2. Check its return code, should be 0 if all services are ok else
-               != 0
+    :expectedresults: hammer ping returns a right return code
+    """
+    result = ssh.command(f'hammer -u {self.foreman_user} -p {self.foreman_password} ping')
+    assert result.stderr == ''
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ping.py:39]: I wonder, how this assert worked for you here, for me
 ```
>       assert result.stderr == ''
E       AssertionError: assert b'' == ''
```

Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ping.py:39]: Here hammer ping doesn't require username and password 
```suggestion
    result = ssh.command('hammer ping')
    assert result.stderr == ''
    assert result.return_code == 0
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ping.py

File: tests/foreman/cli/test_ping.py (hunk 1 above)
Comment (mshriver) [reply tests/foreman/cli/test_ping.py:39]: This is missing a commit! `stderr` does need to be decoded to make this comparison, I just hadn't pushed the commit. Good catch!

Comment (mshriver) [reply tests/foreman/cli/test_ping.py:39]: Ack, just keeping the test functionally the same as it was before. If you're confident in me removing those arguments (I agree, its not necessary), I'll remove them in this PR.

File: tests/foreman/cli/test_ping.py [hunk 2]
```diff
@@ -12,57 +12,47 @@
...
+           != 0
 
-        :steps:
-            1. Execute hammer ping
-            2. Check its return code, should be 0 if all services are ok else
-               != 0
+    :expectedresults: hammer ping returns a right return code
+    """
+    result = ssh.command('hammer resultping')
+    assert result.stderr.decode() == ''
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_ping.py:39]: @mshriver this is a wrong command, it should be as `hammer ping` only :)

And add one more assert/comparison for `result.returncode` after stderr, just an additional check can be covered within the same PR.

Comment (mshriver) [reply tests/foreman/cli/test_ping.py:39]: Its already covered further down in the test. It actually asserts the RC against what it should be given the service status.

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ping.py

File: tests/foreman/cli/test_ping.py (hunk 2 above)
Comment (Gauravtalreja1) [reply tests/foreman/cli/test_ping.py]: Yes, my bad :)

**Review by Gauravtalreja1 — APPROVED** [review]
LGTM 👍

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**